resources/views/volumes/clone.blade.php

```diff
@@ -0,0 +1,195 @@
+                            </div>
+                            <label-trees v-if="restrictFileLabels" :trees="fileLabelTrees" :multiselect="true"
+                                         :allow-select-siblings="true" :allow-select-children="true"
+                                         class="request-labels-well well well-sm"></label-trees>
+                        </div>
+
+                        <div class="checkbox">
+                            <div>
+                                <label>
+                                    <input type="checkbox" id="annotations" v-model="cloneAnnotationLabels">
+                                    Clone annotations</label>
+                            </div>
+                            <div class="form-group{{ $errors->has('clone_annotations') ? ' has-error' : '' }}">
+                                @if($errors->has('clone_annotations'))
+                                    <span class="help-block">{{ $errors->first('clone_annotations') }}</span>
+                                @endif
+                            </div>
+
+                            <div v-if="cloneAnnotationLabels">
+                                <div>
+                                    <label>
+                                        <input type="checkbox" v-if="cloneAnnotationLabels" id="annotationLabel"
+                                               v-model="restrictAnnotationLabels"> Restrict annotation labels (<span
+                                            v-text="selectedAnnotationLabelsCount"></span> labels selected)
+                                    </label>
+                                </div>
+                                <label-trees v-if="cloneAnnotationLabels && restrictAnnotationLabels"
+                                             :trees="annotationLabelTrees"
+                                             :multiselect="true"
+                                             :allow-select-siblings="true" :allow-select-children="true"
+                                             class="request-labels-well well well-sm"></label-trees>
+                            </div>
+                            <div class="form-group{{ $errors->has('only_annotation_labels') ? ' has-error' : '' }}">
+                                @if($errors->has('only_annotation_labels'))
+                                    <span class="help-block">{{ $errors->first('only_annotation_labels') }}</span>
+                                @endif
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <input type="submit" class="btn btn-success pull-right" :disabled="cannotSubmit" value="Clone">
+                    <a class="btn btn-default pull-right" :disabled="loading" type="button"
+                       href="{{\Illuminate\Support\Facades\URL::previous()}}">Cancel</a>
```

Comment:
There is an alias for this facade in Laravel, so you don't have to give the full namespace.

```suggestion
                       href="{{URL::previous()}}">Cancel</a>
```